src/main/cpp/vision/DragonQuest.h

```diff
@@ -47,7 +47,6 @@ class DragonQuest : public DragonDataLogger, public DragonVisionPoseEstimator
         units::angle::degree_t mountingRoll    /// <I> - Roll of camera
```

Comment:
@Tan560 @notcharlie2026 Probably should indicate the default constructor(s) are deleted so we don't inadvertently create a DragonQuest with none of the parameters set properly.